src/OmniSharp.MSBuild/ProjectFile/PackageReference.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using NuGet.Packaging.Core;
+
+namespace OmniSharp.MSBuild.ProjectFile
+{
+    public class PackageReference : IEquatable<PackageReference>
```

Comment:
right now I remember, in the past I used to define my own `PackageReference` on the [old nuget apis](https://github.com/scriptcs/scriptcs/blob/dev/src/ScriptCs.Core/PackageReference.cs) too, for the very same reason you just mentioned